app/components/app-header.hbs

```diff
@@ -1,3 +1,15 @@
 <header class="app-header">
+  <button
```

Comment:
Default template lint would require `type="button"`. Not sure if we want that here as it's really only necessary in a form.

Reply:
Great catch, will add~